white-test-app-frontend/src/app/test-templates/test-templates.component.ts

```diff
@@ -192,7 +192,7 @@ export class NgbdModalNewTest implements OnInit {
   }
 
   openSnackBar() {
-    this.snackBar.open('CTRL+W -> Open Wiki | CTRL+S -> Search synonym', 'close', {
+    this.snackBar.open("SHIFT + ↑  -> Open Wiki | SHIFT + ↓  -> Search synonym", "close", {
```

Comment:
Cala aplikacje mamy po polsku, wiec tu tez bym ten komunikat dal po polsku

Reply:
Odnośniki do stron tez do polskich wersji?

Reply:
hmm, trudne pytanie, a dałoby rade zrobić tak że jak edytuje/tworze test w jezyku polskim, to zapytanie idzie do polskiej wersji i analogicznie angielskiej?

Reply:
ok, zrobiłem to